bld/CLMBuildNamelist.pm

```diff
@@ -5743,7 +5745,21 @@ sub quote_string {
       $str = "\'$str\'";
    }
    return $str;
- }
+}
+
+#-------------------------------------------------------------------------------
+
+sub remove_newlines {
```

Comment:
Other than both of us (Erik and myself) being puzzled by the fact that there are preexisting examples of fincl statements containing newlines that have never caused trouble and agreeing that we should theoretically try to comprehend why the MEGAN ones caused trouble....

I am approving this PR.

Reply:
...possibly open a low priority issue about it?

Reply:
OK, I made an issue for this #3208